README.md

```diff
@@ -90,8 +90,6 @@ hermeto fetch-deps \
 
 The `fetch-deps` command fetches your project's dependencies and stores them on your disk. Hermeto also produces a detailed SBOM containing information about all the project's components and packages. You can find the SBOM in the output directory.
 
-See [docs/usage.md](docs/usage.md) for a more detailed, practical example of Hermeto usage.
```

Comment:
[multiple-occurrences] I feel like we're losing some information here by not referencing the usage docs. Can we convert these links to just plain text paths enclosed in the `code block` to at least retain the reference?